js-packages/@prestojs/ui-antd/src/widgets/RangeWidget.tsx

```diff
@@ -16,14 +16,33 @@ function RangeWidget<FieldValue, T extends HTMLElement, P>(
     props: RangeWidgetProps<FieldValue, T, P>,
     ref: any
 ): React.ReactElement {
-    const { lowerInput, upperInput, separator = '-', inputWidget: InputWidget, ...rest } = props;
-    const { lowerRef, upperRef } = ref;
+    const {
+        lowerInput,
+        upperInput,
+        separator = '-',
+        inputWidget: InputWidget,
+        input: { value = {} as Record<string, FieldValue>, onChange },
+        ...rest
+    } = props;
+    const { lowerRef, upperRef } = ref || {};
     return (
         <>
             <Input.Group compact>
```

Comment:
add className `className="presto-range-widget-wrapper"`

actually should also support `className` prop at top level I guess, eg. `className={cx("presto-range-widget-wrapper", className)}`